test/modules/sungeun/init/printModuleInitOrder.good

```diff
@@ -11,7 +11,6 @@ Initializing Modules:
      ChapelStringLiterals
     MemConsistency
     Atomics
-    AtomicsCommon
```

Comment:
Was there a use statement that got removed?  I'm confused as to why this was necessary